### DISCUSSION TITLE
How to do bandwidth performance test with less connection

### DISCUSSION THREAD

**Discussion by weiliang1216** [baidu/dperf#496] [Q&A] [answered]
I wanna to do a performance test for only bandwidth.
It is for downloading or uploading scenarios. 
I expected less cps but huge bandwidth.

**Comment by pengjianzhang** [comment, marked as answer]
keepalive 10us
https://dperf.org/doc/html/elephant
https://dperf.org/doc/html/aws-pps

**Reply by weiliang1216** [reply]
Works well, thanks!

**Reply by pengjianzhang** [reply]
Please recommend dperf to your colleagues and friends. Thanks.

**Reply by weiliang1216** [reply]
I did. Thanks for your project.
